bindings/experimental/distrdf/python/DistRDF/Node.py

```diff
@@ -549,273 +513,33 @@ def _get_clustered_ranges(self, treename, filelist,
                 ],  # type: list[str]
                 friend_info  # type: FriendInfo
             )  # type: collections.namedtuple
-            for clusters in _n_even_chunks(clustersinfiles, self.npartitions)
+            for clusters in _n_even_chunks(clustersinfiles, self._headnode.npartitions)
         ]
 
         logger.debug("Created following clustered ranges:\n%s",
                      "\n\n".join(map(str, clustered_ranges)))
 
         return clustered_ranges
 
-    def build_ranges(self):
-        """
-        Define two type of ranges based on the arguments passed to the
-        RDataFrame head node.
+    def _get_filelist(self, files):
```

Comment:
Is this never called?

Reply:
Indeed, the logic to return always a python list of strings has been inlined in `TreeHeadNode.get_inputfiles`. I forgot to remove this function in this commit, it is done later. I can make a fixup and do it in this commit instead